migration_toolkit_for_containers/troubleshooting-mtc.adoc

```diff
@@ -42,5 +43,18 @@ include::modules/migration-error-messages.adoc[leveloffset=+2]
 
 You can roll back a migration by using the {mtc-short} web console or the CLI.
 
+You can also roll back a migration manually.
```

Comment:
Since the procedure for this is a little ways down the page, might consider linking to it from this line so the reader can find it easily if they want to go that route.